### PR TITLE
updated mutations and getitem websiteId

### PR DESCRIPTION
**Pull request by rdoughty**

### CODE REVIEW

File: src/components/Pages/Collection/index.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ const Collection = ({ id, location }) => {
         `
     } else {
       query = `mutation {
-          replaceDefaultImage(input: {
+          saveDefaultImageForWebsite(input: {
```
Comment (SteveMattison) [inline src/components/Pages/Collection/index.js:110]: The parameters changed here too.  
The new parameters are:  itemId, defaultFilePath, and objectFileGroupId

File: src/components/Pages/Collection/index.js [hunk 2]
```diff
@@ -97,7 +97,7 @@ const Collection = ({ id, location }) => {
     let query = ''
     if (typeof generalPartiallyDigitized !== 'undefined') {
       query = `mutation {
-          replacePartiallyDigitized(input: {
+          savePartiallyDigitizedForWebsite(input: {
```
Comment (SteveMattison) [inline src/components/Pages/Collection/index.js:100]: Parameters changed here.  New parameters are:  itemId and partiallyDigitized.

**Review by SteveMattison — CHANGES_REQUESTED** [review]

File: src/components/Pages/Collection/index.js (hunk 1 above)
Comment (rdoughty) [reply src/components/Pages/Collection/index.js:110]: thanks didn't see that the inputs changed

File: src/components/Pages/Collection/index.js (hunk 2 above)
Comment (rdoughty) [reply src/components/Pages/Collection/index.js:100]: thanks didn't see that the inputs changed

**Review by SteveMattison — CHANGES_REQUESTED** [review]
I see that the "input" layer is still in the calling signature.  We have eliminated the "input" to make all calls more consistent.  Please pass the input parameters directly, without using "input".  Thanks

This is true for calling both mutations.

**Review by danthewolfe — APPROVED** [review]

**Review by danthewolfe — APPROVED** [review]

**Review by SteveMattison — APPROVED** [review]

**Pull request merged**